src/seedu/addressbook/data/person/Fees.java

```diff
@@ -9,13 +9,21 @@
  */
 public class Fees implements Printable {
     public static final String EXAMPLE = "$3560.98";
-    public static final String MESSAGE_FEES_CONSTRAINTS = "Person's fees should have 2 decimal places.";
+    public static final String MESSAGE_FEES_CONSTRAINTS = "Person's fees should be positive and have 2 decimal places.";
     public static final String MESSAGE_DATE_CONSTRAINTS = "Due date for fees should be in DD-MM-YYYY.";
     public static final String FEES_VALIDATION_REGEX = "[0-9]+([,.][0-9]{1,2})?";
     public static final String DATE_VALIDATION_REGEX = "[0-9]{1,2}-[0-9]{1,2}-[0-9]{4}";
 
     public final String value;
     public final String duedate;
+    private boolean isEdited = false;
+
+    //TODO hide the fees when it's first initialised as 00-00-0000
+    // new command for when they paid fees, fully, thus reinitiliasing as 00-00-0000
+    // so that it doesnt fail valid date check in edit fees
+    // therefore fees will always either be 00-00-0000 or a valid date,
+    // and will always be hidden when its 00-00-0000
+    // this way is easier as dont need to change all the adapted person and stuff
 
```

Comment:
What's the point of TODO if this is the final iteration?

Reply:
Oh it's done alr sorry let me remove it